<?xml version="1.0" encoding="UTF-8"?>
<testrun name="DisasterAppActivityTest (1)" project="DisasterAppTest" tests="2" started="2" failures="1" errors="1" ignored="0">
  <testsuite name="Phone_Large [emulator-5554]" time="232.31">
    <testsuite name="com.riis.test.DisasterAppActivityTest" time="232.31">
      <testcase name="testCreateContactButtonIntent " classname="com.riis.test.DisasterAppActivityTest" time="6.557">
        <failure>junit.framework.AssertionFailedError: expected:&lt;1&gt; but was:&lt;0&gt;&#13;
at com.riis.test.DisasterAppActivityTest.testCreateContactButtonIntent(DisasterAppActivityTest.java:39)&#13;
at java.lang.reflect.Method.invokeNative(Native Method)&#13;
at android.test.InstrumentationTestCase.runMethod(InstrumentationTestCase.java:214)&#13;
at android.test.InstrumentationTestCase.runTest(InstrumentationTestCase.java:199)&#13;
at android.test.ActivityInstrumentationTestCase2.runTest(ActivityInstrumentationTestCase2.java:192)&#13;
at android.test.AndroidTestRunner.runTest(AndroidTestRunner.java:190)&#13;
at android.test.AndroidTestRunner.runTest(AndroidTestRunner.java:175)&#13;
at android.test.InstrumentationTestRunner.onStart(InstrumentationTestRunner.java:555)&#13;
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:1661)&#13;

</failure>
      </testcase>
      <testcase name="testCreateContactScreenButtonExists " classname="com.riis.test.DisasterAppActivityTest" time="225.701">
        <error>Test failed to run to completion. Reason: 'Instrumentation run failed due to 'Process crashed.''. Check device logcat for details
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testrun>
